openhtf/__init__.py

```diff
@@ -37,11 +38,35 @@
 
 FLAGS = gflags.FLAGS
 FLAGS(sys.argv)
-
+gflags.DEFINE_string('log_dir', '/tmp', 'dir for log files')
+gflags.DEFINE_enum('log_output', 'console', ['console','file','both'], 'output option')
+gflags.DEFINE_enum('level', 'info', ['debug', 'info', 'warning', 'error', 'critical'], 'log level')
+
+def Setup_logger():
+    htflogger = logging.getLogger('HTFTest')
```

Comment:
I think we're going to want to use the htflogger (possibly with a re-name for clarity) so we get log messages in our test record in addition to the normal logfile and stdout/stderror.